From d29e034f3c929e0058fcdea44e2db0af9ba6011c Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 24 Oct 2018 14:57:06 +0800
Subject: [PATCH] arm64: dts: rockchip: parse usb_pcie_grf for combphy driver
 for rk1808

Innosilicon combphy need release link reset grant
when finishing PLL lock, so we need the driver to
control usb_pcie_grf.

Change-Id: If429629b39d1f68a0fdcb24c6b639f84d513aee5
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index ff79aa034563..ec36f2a81002 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -268,6 +268,11 @@
 		};
 	};
 
+	usb_pcie_grf: syscon@fe040000 {
+		compatible = "rockchip,usb-pcie-grf", "syscon";
+		reg = <0x0 0xfe040000 0x0 0x1000>;
+	};
+
 	coregrf: syscon@fe050000 {
 		compatible = "rockchip,rk1808-coregrf", "syscon", "simple-mfd";
 		reg = <0x0 0xfe050000 0x0 0x1000>;
@@ -418,6 +423,7 @@
 		reset-names = "otg-rst", "combphy-por",
 			      "combphy-apb", "combphy-pipe";
 		rockchip,combphygrf = <&combphy_grf>;
+		rockchip,usbpciegrf = <&usb_pcie_grf>;
 		status = "disabled";
 	};
 
-- 
2.35.3

